config/homebin/is_utility_installed

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+VVV_CONFIG=/vagrant/config.yml
+
+utilities=$(shyaml get-values "utilities.${1}" 2> /dev/null < ${VVV_CONFIG})
```

Comment:
A micro-optimisation would be to check for the folder since we know the name of the utility, but I doubt that'll save very much time

Reply:
I'd say it's on the ms range, right?